### PR TITLE
Migrate remaning commigs from scala3doc repo

### PR DESCRIPTION
**Pull request by romanowski**

### CODE REVIEW

**Comment by abgruszecki** [comment]
@romanowski can also we fix the indentation to always be 2 spaces instead of sometimes 2, sometimes 4?

File: scala3doc/resources/dotty_res/styles/diagram.css [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
-    height: 80%;
+  width: 100%;
+  height: 80%;
 }
+
+.diagram-class a {
+  text-decoration: underline;
+  color: #FFF;
+}
+
```
Comment (TheElectronWill) [inline scala3doc/resources/dotty_res/styles/diagram.css:26]:
```suggestion
.diagram-class a:hover {
  color: #BFE7F3;
}
```
This change has been skipped. The rest LGTM

**Review by abgruszecki — APPROVED** [review]
Everything LGTM, as far as I can tell.

I rendered the documentation locally and everything looked good as well.

**Comment by abgruszecki** [comment]
OK, I think we can merge this, just to avoid conflicts piling up. If there are any remaining issues, we can fix them afterwards.

**Comment by bishabosha** [comment]
merging because it only affects the scala3doc code

**Pull request merged**